From 111068d47cb2b7a45719bb7f472dbfd4050e2244 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 25 Nov 2015 10:19:10 +0000
Subject: [PATCH 0625/1000] MANTIS 39108 Config service 1.6.1 - MANTIS 39108:
 Fix traces in Windows event log - No longer write to Windopws event log.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@929 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/CfgXMgrB/inc/CfgXMgrB_def.h          | 13 +++++++++++--
 .../Massai/cpp/CfgXMgrB/inc/massaiCfgXB_def.h       | 13 +++++++++++--
 2 files changed, 22 insertions(+), 4 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/CfgXMgrB_def.h b/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/CfgXMgrB_def.h
index 093400bc..efcb8cb6 100644
--- a/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/CfgXMgrB_def.h
+++ b/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/CfgXMgrB_def.h
@@ -456,8 +456,17 @@
 //! - MANTIS 39137: Platform CfgXMgrB: Introduce new command to set
 //!   meta information for existing configuration values.
 //! - new method setMeta() for text, text-list, number, and selection.
-#define FILE_VER_BIN 1,6,0,0
-#define FILE_VER_DTS "1.6.0.0"
+// #define FILE_VER_BIN 1,6,0,0
+// #define FILE_VER_DTS "1.6.0.0"
+//
+//! @version 1.6.1
+//! @date    25.11.2015
+//! @author  JK
+//! @par     Change:
+//! - MANTIS 39108: Fix traces in Windows event log
+//! - No longer write to Windopws event log.
+#define FILE_VER_BIN 1,6,1,0
+#define FILE_VER_DTS "1.6.1.0"
 //
 
 
diff --git a/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/massaiCfgXB_def.h b/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/massaiCfgXB_def.h
index d6594971..e49f556b 100644
--- a/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/massaiCfgXB_def.h
+++ b/SelfServiceCommon/Massai/cpp/CfgXMgrB/inc/massaiCfgXB_def.h
@@ -447,8 +447,17 @@
 //! - MANTIS 39137: Platform CfgXMgrB: Introduce new command to set
 //!   meta information for existing configuration values.
 //! - new method setMeta() for text, text-list, number, and selection.
-#define FILE_VER_BIN 1,6,0,0
-#define FILE_VER_DTS "1.6.0.0"
+// #define FILE_VER_BIN 1,6,0,0
+// #define FILE_VER_DTS "1.6.0.0"
+//
+//! @version 1.6.1
+//! @date    25.11.2015
+//! @author  JK
+//! @par     Change:
+//! - MANTIS 39108: Fix traces in Windows event log
+//! - No longer write to Windopws event log.
+#define FILE_VER_BIN 1,6,1,0
+#define FILE_VER_DTS "1.6.1.0"
 //
 
 //-------------------------------------------------------------------------
-- 
2.41.0.windows.1

